Gemfile

```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'aws-sdk', '~> 1.34.0', :require => nil
+gem 'addressable', '~> 2.6', require: nil
```

Comment:
This will make `addressable` a transitive dependency of every app that uses `asset_cloud`. I'm usually cautious about adding dependencies to a gem if we can rely on stdlib instead.

While `CGI.escape` is not an option here, we do have `ERB::Util.url_escape` which is a part of stdlib. AFAICT, it does exactly what we need here without adding to the dependency tree.

```ruby
ERB::Util.url_encode('hello[world]')
#=> "hello%5Bworld%5D"
```